src/ethereum_test_tools/rpc/rpc.py

```diff
@@ -0,0 +1,111 @@
+"""
+JSON-RPC methods and helper functions for EEST consume based hive simulators.
+"""
+
+from typing import Dict, List, Literal, Union
+
+import requests
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+from ethereum_test_tools import Address
+
+BlockNumberType = Union[int, Literal["latest", "earliest", "pending"]]
+
+
+class BaseRPC:
```

Comment:
We could inherit python's `abc.ABC` abstract base class library class to express/enforce that `BaseRPC` can never be instantiated. If we add abstract methods we can mark them as such using the `abc.abstractmethod` decorator. We don't widely use this yet, but perhaps we should.

```suggestion
from abc import ABC

class BaseRPC(ABC):
```
This will break tox ))